product_internal_reference_generator/tests/test_product_internal_reference_generator.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2023 Ooops - Ilyas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import Form, TransactionCase
+
+
+class TestProductInternalReferenceGenerator(TransactionCase):
```

Comment:
You could maybe use `BaseCommon`class from https://github.com/odoo/odoo/blob/a877336b4d007c06c9759e4c8a302038bae20a26/odoo/addons/base/tests/common.py#L19

As it enhances tests performances.